ports/bare-arm/Makefile

```diff
@@ -3,6 +3,7 @@ include ../../py/mkenv.mk
 
 # Include py core make definitions.
 include $(TOP)/py/py.mk
+include $(TOP)/extmod/extmod.mk
```

Comment:
Does bare-arm need anything from extmod?

Reply:
I guess not. I did not realize it is basically the same as the `minimal` port.